perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -36,16 +35,26 @@ MOUNT_POINT=gcs
 # The VM will itself exit if the gcsfuse mount fails.
 gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 
-# Executing perf tests
-chmod +x run_load_test_and_fetch_metrics.sh
-./run_load_test_and_fetch_metrics.sh
+chmod +x perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
+./perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
 # Copying gcsfuse logs to bucket
 gsutil -m cp $LOG_FILE gs://gcs-fuse-dashboard-fio/fio-gcsfuse-logs/
 
 # Deleting logs older than 10 days
-python3 utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
+python3 perfmetrics/scripts/utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
 
 # ls_metrics test
-cd "./ls_metrics"
-chmod +x run_ls_benchmark.sh
-./run_ls_benchmark.sh
+chmod +x perfmetrics/scripts/run_ls_benchmark.sh
+./perfmetrics/scripts/run_ls_benchmark.sh
+
+sudo umount gcs
+
+# Executing implicitdir integration tests for testbucket flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --testbucket=gcsfuse-implicitdir-tests
+# Mounting gcsfuse for running integration tests for mountedDirectory flag
+BUCKET_NAME=gcsfuse-implicitdir-tests
+MOUNT_POINT=gcs
+# The VM will itself exit if the gcsfuse mount fails.
+gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
+# Executing implicitdir integration tests for mountedDirectory flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --mountedDirectory=../../../$MOUNT_POINT
```

Comment:
instead of this --mountedDirectory=../../../$MOUNT_POINT we can do
fullPath=$(readlink -f $MOUNT_POINT)
--mountedDirectory=$fullPath